### PR TITLE
[MODINVOSTO-170] - Release v5.7.0

### PR DESCRIPTION
**Pull request by azizbekxm**
## Purpose

https://issues.folio.org/browse/MODINVOSTO-170
## Pre-Merge Checklist:
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all the appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?
- Did you modify code to call some additional endpoints?
  - [ ] If so, do you check that necessary module permission added in ModuleDescriptor-template.yaml file?

Ideally, all the PRs involved in breaking changes would be merged on the same day
to avoid breaking the folio-testing environment.
Communication is paramount if that is to be achieved,
especially as the number of inter-module and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems,
ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,4 +1,26 @@
...
+
+### New APIs versions
+* Requires `PUT v5.7.0` extra fiscalYearId in DTO
+
+### Features
+* Update to Java 17 mod-invoice-storage ([MODINVOSTO-166](https://issues.folio.org/browse/MODINVOSTO-166))
+* Create invoice line numbers based on new invoice field ([MODINVOSTO-162](https://issues.folio.org/browse/MODINVOSTO-162))
+* Update models schema with new fields for ftp configuration ([MODINVOICE-455](https://issues.folio.org/browse/MODINVOICE-455))
+* Update dependent raml-util ([MODINVOSTO-154](https://issues.folio.org/browse/MODINVOSTO-154))
+* Update DTO schema with adding fiscalYearId to PUT payload ([MODINVOSTO-144](https://issues.folio.org/browse/MODINVOSTO-144))
```
Comment (Abdulkhakimov) [inline NEWS.md:16]: Please follow release notes as it was

Comment (azizbekxm) [reply NEWS.md:16]: I read this documentation https://wiki.folio.org/pages/viewpage.action?pageId=96404780. Looks like there is no documentation to follow common documentation for across all teams. 

I will try to follow existed prs.

File: NEWS.md [hunk 2]
```diff
@@ -1,4 +1,30 @@
...
+### New APIs versions
+* Requires `PUT v5.7.0` extra fiscalYearId in DTO
+
+### Stories
+* ([MODINVOSTO-166](https://issues.folio.org/browse/MODINVOSTO-166)) - Update to Java 17 mod-invoice-storage 
+* ([MODINVOSTO-162](https://issues.folio.org/browse/MODINVOSTO-162)) - Create invoice line numbers based on new invoice field
+* ([MODINVOICE-455](https://issues.folio.org/browse/MODINVOICE-455)) - Update models schema with new fields for ftp configuration
+* ([MODINVOSTO-154](https://issues.folio.org/browse/MODINVOSTO-154)) - Update dependent raml-util
+* ([MODINVOSTO-144](https://issues.folio.org/browse/MODINVOSTO-144)) - Update DTO schema with adding fiscalYearId to PUT payload
+* ([FOLIO-3678](https://issues.folio.org/browse/FOLIO-3678)) - Use GitHub Workflows api-lint and api-schema-lint and api-doc
```
Comment (SerhiiNosko) [inline NEWS.md:21]: brackets not needed, please how stories below formatted

File: NEWS.md [hunk 3]
```diff
@@ -1,4 +1,30 @@
...
+
+### Bug fixes
+* ([MODORDSTOR-343](https://issues.folio.org/browse/MODORDSTOR-343)) - Duplicated PO lines have the same numbers related to newly created duplicated order
+
+### Dependencies
```
Comment (Dmitriy-Butramyou) [inline NEWS.md:26]: I think we can delete this and also Breaking changes.

**Review by Dmitriy-Butramyou — APPROVED** [review]

File: NEWS.md [hunk 4]
```diff
@@ -1,4 +1,30 @@
...
+## 5.7.0 - Released (Poppy R2 2023)
+
+The primary focus of update DTO scheme and upgraded to java 17 version 
+
+[Full Changelog](https://github.com/folio-org/mod-invoice-storage/compare/v5.6.0...v5.7.0)
+
+### Breaking changes
+* ([MODINVOICE-466](https://issues.folio.org/browse/MODINVOICE-466)) - Update DTO schema with adding fiscalYearId to PUT payload
+
+### New APIs versions
```
Comment (Dmitriy-Butramyou) [inline NEWS.md:12]: As I see it, we have never used this section.

File: NEWS.md (hunk 2 above)
Comment (azizbekxm) [reply NEWS.md]: reformatted

File: NEWS.md (hunk 3 above)
Comment (SerhiiNosko) [reply NEWS.md:21]: it can be useful info, whay we need to delete it?

File: NEWS.md (hunk 4 above)
Comment (SerhiiNosko) [reply NEWS.md]: it can be useful info, whay we need to delete it?

File: NEWS.md (hunk 3 above)
Comment (azizbekxm) [reply NEWS.md:21]: image
other modules used this style, may be we consider to add extra sections

**Review by SerhiiNosko — APPROVED** [review]

**Pull request closed without merging**